tests/meta/test_invoice.py

```diff
@@ -0,0 +1,145 @@
+import mock
+import pytest
+from tests.store_helpers import StoreHelpers
+from cg.meta.invoice import InvoiceAPI
+from tests.mocks.limsmock import MockLimsAPI
+from cg.store import Store
+from cg.constants.invoice import CostCenters, CustomerNames
+from cg.constants.sequencing import RecordType
+from cg.constants.priority import PriorityTerms
+from cg.store.models import Pool, Sample
+
+from cg.models.invoice.invoice import InvoiceInfo
+
+
+@pytest.fixture(name="get_invoice_api_sample")
+def fixture_invoice_api_sample(
```

Comment:
Define the fixture in the relevant conftest.py and import it in the test function